<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myIDE</class>
 <widget class="QMainWindow" name="myIDE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myIDE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableMetadata">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>651</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Serif</family>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Serif</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Información de la búsqueda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TEST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Serif</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPicture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>421</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>641</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>570</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="enterButton">
       <property name="font">
        <font>
         <family>Liberation Serif</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="font">
        <font>
         <family>Liberation Serif</family>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>290</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Serif</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>MyIDE Editor</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
